src/components/ContactsList/ContactHeaderRow.jsx

```diff
@@ -1,9 +1,45 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import { withStyles } from '@material-ui/core/styles'
 
-const ContactHeaderRow = props => <div className="divider">{props.header}</div>
+import Paper from 'cozy-ui/transpiled/react/Paper'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+
+const customStyles = theme => ({
+  paper: {
+    position: 'sticky',
+    zIndex: 1,
+    top: 0,
+    backgroundColor: 'var(--defaultBackgroundColor)',
+    height: '2rem',
+    display: 'flex',
+    alignItems: 'center',
+    textIndent: '2rem',
+    textTransform: 'uppercase',
```

Comment:
Those font styling attributes should be in the Typography here no ? Why do we have to add custom styles, and not use a variant of Typography ?

Reply:
Ok pour déplacer dans le Typography.
C'est un cas très spécifique d'affichage d'une barre horizontal avec une lettre à l’intérieur, les variants de typography ne permette pas ce résultat à ma connaissance

![image](https://user-images.githubusercontent.com/67680939/106761285-f46c4180-6634-11eb-8183-06dae9d43538.png)

Reply:
Le ListSubheader semble tout indiqué ici : voir mode mobile de https://docs.cozy.io/cozy-ui/react/#!/NavigationList

Reply:
ah bien vu merci !